bin/index.js

```diff
@@ -75,6 +75,12 @@ yargs.command('getreg <playerName> [section] [key]', 'Get registry values', (yar
 // edit registry
 yargs.command('setreg <playerName> <section> <key> <value>', 'Edit registry values', (yargs) => {positionals.setRegPositional(yargs)}, (argv) => {handlers.editReg(argv)});
 
+// set power save
+yargs.command('setpowersave <playerName> [enabled] [connector] [device]', 'Set the video connector power save', (yargs) => {positionals.setPowerSavePositional(yargs)}, (argv) => {handlers.setPowerSave(argv)});
+
+// get power save
+yargs.command('getpowersave <playerName> [connector] [device]', 'Get the video connector power save', (yargs) => {positionals.getPowerSavePositional(yargs)}, (argv) => {handlers.getPowerSave(argv)});
```

Comment:
Parameters are marked <>, or required, in the readme. But here they are marked [], or optional. I think they should all be required for set, but either way it should be consistent.

Reply:
@lherlein Required as opposed to having a default value associated with them? 

enabled (default: true)
connector (default: HDMI)
device (default: 0)

AFAICS from our REST API DWS Documentation, we only support HDMI 0 for power save.

Reply:
AFAIK positionals cannot have defaults. It would be nearly impossible to parse. This is another reason that the movement to options on v2 is so nice